xbmc/video/VideoDatabase.cpp

```diff
@@ -6669,11 +6669,40 @@ void CVideoDatabase::UpdateTables(int iVersion)
 
     m_pDS->exec("DELETE FROM episode WHERE idSeason NOT IN (SELECT idSeason from seasons)");
   }
+
+  if (iVersion < 134)
+  {
+    // renumber itemType to free the value 0 for nodes navigation
+    // former value 0 for versions becomes 1
+    // former value 1 for extras becomes 2
+
+    constexpr int VIDEOASSETTYPE_VERSION_OLD = 0;
+    constexpr int VIDEOASSETTYPE_EXTRA_OLD = 1;
```

Comment:
`static constexpr` - you don't want the constants created on every function invocation.